README.md

```diff
@@ -46,6 +46,13 @@ The project is to determine the basic knowledge one has in git,HTML5,CSS,CSS Gri
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 
+## 🚀 Live Demo <a name="live-demo"></a>
+
+https://markyegon7.github.io/Mark-Portfolio/
```

Comment:
- [ Optional ] Kindly I recommend you to use link to GitHub page just like this [Live Demo](https://markyegon7.github.io/Mark-Portfolio/) Just provide the link directly to the README file.